### ISSUE TITLE
scriptsBlocked message is not intuitive

### ISSUE THREAD

**Issue by luixxiul** [brave/browser-laptop#7377]
**Describe the issue you encountered:** scriptBlocked message, "Do you want to allow scripts on {{site}} from these sources?", is confusing and not intuitive.

It looks like as if they were origins which had been already allowed.

<img width="473" alt="screenshot 2017-02-25 10 56 03" src="https://cloud.githubusercontent.com/assets/3362943/23327067/28c6f480-fb49-11e6-883d-4eaeda2b6aa1.png">

In that case I am not quite sure which origin will be actually allowed and denied (I can understand by thinking carefully, but not 100% sure).

- Brave Version (revision SHA): d9234b5

- Steps to reproduce: Open https://github.com, click the noScriptInfo icon

- Actual result: see above

- Expected result: I think the allowed origins should only be checked, which means that the noScriptInfo should be configuration dialog same as the site shield settings;

    ````
    allowed == checked 
    denied == unchecked
    ````
    For me that looks natural.

- Any related issues: #6431

**Comment by luixxiul** [comment]
Updated the post.
